core/src/main/scala/org/apache/spark/scheduler/ReplayListenerBus.scala

```diff
@@ -118,8 +118,6 @@ private[spark] class ReplayListenerBus extends SparkListenerBus with Logging {
       case e: HaltReplayException =>
         // Just stop replay.
       case _: EOFException if maybeTruncated =>
-      case _: IOException if maybeTruncated =>
```

Comment:
Can this still happen for non-zstd compression though?

Reply:
This was added for zstd incomplete frame reading issue. But after this change, that issue is no longer happens. Yes. we can keep as it is.